ts/components/messages/MessageList.tsx

```diff
@@ -227,6 +227,7 @@ class MessageList extends React.Component<Props, State> {
     const potService = this.props.servicesById[meta.sender_service_id];
```

Comment:
It works!
Anyaway we could rewrite it to improve logic and readability.
The condition to satisfied should be: we have to show a loading item if either service or message are in loading state.

```typescript
const potService = this.props.servicesById[meta.sender_service_id];
    const isServiceLoading = potService
      ? pot.isNone(potMessage) && pot.isLoading(potService) // is none loading
      : false;
    if (isServiceLoading || pot.isLoading(potMessage)) {
      return MessageListItemPlaceholder;
    }
```
Could it be better?

Reply:
yes,
but this: 

> pot.isNone(potMessage) && pot.isLoading(potService) // is none loading 

it should be corrected like this: `pot.isNone(potService) && pot.isLoading(potService) // is none loading`